bin/zsnap

```diff
@@ -39,15 +40,27 @@ else:
 set_verbose(opts.verbose)
 
 snapshot_prefix = opts.prefix
-snapshot_postfix = lambda: time.strftime(opts.timeformat)
+snapshot_postfix = lambda: time.strftime(opts.timeformat,time.gmtime())
+
+snapshot_properties = {}
+
+if opts.property:
+	split = opts.property.split('=')
```

Comment:
This should be ...split('=', 1), otherwise property values that contain an equals will fail.